src/appShell/App/PortalFooter.tsx

```diff
@@ -1,22 +1,61 @@
 import * as React from 'react';
+// tslint:disable-next-line:no-import-side-effect
+import './footer.scss';
+import AppConfig from "appConfig";
+import _ from 'lodash';
 
 export default class PortalFooter extends React.Component<{}, {}> {
-
-    render(){
-        return <div id="footer">
-            <a href="http://cbioportal.org">cBioPortal</a>
-            <span id="footer-span-version"> Version 1.14.0-SNAPSHOT</span>
-            <span className="pipeSeperator">|</span><a href="http://www.mskcc.org/mskcc/html/44.cfm">MSKCC</a><span className="pipeSeperator">|</span>
-            <a href="http://cancergenome.nih.gov/">TCGA</a>
-            <br/>
-            Questions and feedback: <span className="mailme" title="Contact us">cbioportal at googlegroups dot com</span>
-            <span className="pipeSeperator">|</span>
-            <a target="_blank" href="http://groups.google.com/group/cbioportal">User discussion group</a>
-            <span className="pipeSeperator">|</span>
-            <a target="_blank" href="https://github.com/cBioPortal/">
-            <i className="fa fa-github" aria-hidden="true"></i> GitHub</a>
-        </div>
+    render() {
+            if (AppConfig.serverConfig.skin_footer && !_.isEmpty(AppConfig.serverConfig.skin_footer)) {
+                return (
+                    <div id="footer" dangerouslySetInnerHTML={{__html: "<a href='http://www.cbioportal.org'>cBioPortal</a>" + AppConfig.serverConfig.skin_footer}}>
```

Comment:
I don't know how important this is but there's a little more nuance to the legacy foot configuration.  
 
/Users/aaronlisman/WebstormProjects/cbioportal/portal/src/main/webapp/WEB-INF/jsp/global/footer.jsp

Reply:
@alisman good catch -- @jjgao should all these properties of the old footer be kept?

Reply:
This is the old one: https://github.com/cBioPortal/cbioportal/blob/master/portal/src/main/webapp/WEB-INF/jsp/global/footer.jsp

Reply:
i guess main thing is `getEmailContact()` - maybe good to continue to support that. It might be easiest to only show this new footer for the public facing portal only